actors/runtime/runtime.go

```diff
@@ -213,25 +197,19 @@ type Syscalls interface {
 // the return, in particular whether it has been serialized to bytes or just passed through.
 // Production code is expected to de/serialize, but test and other code may pass the value straight through.
 type SendReturn interface {
-	Into(CBORUnmarshaler) error
-}
-
-// Provides (minimal) tracing facilities to actor code.
-type TraceSpan interface {
-	// Ends the span
-	End()
+	Into(cbor.Unmarshaler) error
 }
 
 // StateHandle provides mutable, exclusive access to actor state.
 type StateHandle interface {
 	// Create initializes the state object.
 	// This is only valid in a constructor function and when the state has not yet been initialized.
-	Create(obj CBORMarshaler)
+	Create(obj cbor.Marshaler)
```

Comment:
Similar for naming: StateCreate, StateRead, StateTransact

The Message and Syscalls embeds are fine without longer names, though.